phaser/phase.py

```diff
@@ -78,27 +149,6 @@ def process_exception(self, exc, step, row):
                     raise PhaserError(f"Unknown error policy '{self.error_policy}'") from exc
 
 
-class DataFramePhase(PhaseBase):
-    def __init__(self, name, context=None, error_policy=None):
-        super().__init__(name, context=context, error_policy=error_policy)
-        self.df_data = None
-
-    def run(self):
-        self.df_data = self.df_transform(self.df_data)
-        return self.df_data.to_dict('records')
-
-    @abstractmethod
-    def df_transform(self, df_data):
-        """ The df_transform method is implemented in subclasses of DataFramePhase.  Significant reshaping can be
-        done because data is not processed row-by-row, and row numbers are not reported on in error reporting.
-        """
-        raise PhaserError("Subclass DataFramePhase and return a new dataframe in the 'df_transform' method")
-
-    def load_data(self, data):
-        # Overrides the regular load_data because we want to convert incoming data to DataFrame.
-        self.df_data = pd.DataFrame(data)
-
-
 class ReshapePhase(PhaseBase):
```

Comment:
Is `ReshapePhase` even needed as a separate class anymore?

Reply:
Ah. It still needs load and whatnot (as claimed in the `run_steps` function) but it doesn't need the `do_columns_stuff` and consistency check.  Got it.